test/functional/feature_rbf.py

```diff
@@ -67,14 +67,15 @@ def set_test_params(self):
         self.num_nodes = 2
         self.extra_args = [
             [
+                "-acceptnonstdtxn=1",
```

Comment:
I can confirm this is needed to pass the test, but I'm confused why.

Reply:
The test spends from anyone-can-spend "padded" scriptPubKeys such as `CScript([b'a' * 35])`